CODEOWNERS

```diff
@@ -305,6 +306,7 @@
 /drivers/wifi/winc1500/                   @kludentwo
 /drivers/virtualization/		  @tbursztyka
 /dts/arc/                                 @abrodkin @ruuddw @iriszzw
+/dts/arm/acsip/                           @NorthernDean
```

Comment:
@galak Can't recall the conclusion of the dev review. Is that the expected way ?